lib/namespaces.js

```diff
@@ -1,5 +1,6 @@
 const namespace = require('@rdfjs/namespace')
 
 module.exports = {
-  code: namespace('https://code.described.at/')
+  code: namespace('https://code.described.at/'),
+  hydra: namespace('http://www.w3.org/ns/hydra/core#')
```

Comment:
We already have a dependency on `@tpluscode/rdf-ns-builders` which exports this NS

Reply:
The namespaces file in a project gives me an overview of namespaces used in a project. It allows me to copy & paste it when I do some simple curls. In general, I prefer to have them explicitly mentioned in the project.

Reply:
Fair enough but FYI the type declarations actually explicitly mention URLs if you ever need to copy/paste. In you IDE you can navigate to `hydra.Class` for example and in the `.d.ts` file you'll find 

```
"Class": NamedNode<'http://www.w3.org/ns/hydra/core#Class'>;
```